server/ml_model/model_base.py

```diff
@@ -0,0 +1,29 @@
+import io
+import json
+
+import torchvision.transforms as transforms
+from PIL import Image
+from torchvision import models
+
+imagenet_class_index = json.load(open('ml_model/imagenet_class_index.json'))
+model = models.densenet121(pretrained=True)
+model.eval()
+
+
+def transform_image(image_bytes):
+    my_transforms = transforms.Compose([transforms.Resize(255),
```

Comment:
Please, don't use magic numbers! Other team members should know, what means number 255 in your code.
Please, add new variable with appropriate name and will be great if you add comment to variable

Reply:
Thank you for the code review! You're absolutely right.
The thing is, I just copy-pasted this piece of code, so I have no idea what those numbers mean.